tachyon/zk/plonk/vanishing/circuit_polynomial_builder.h

```diff
@@ -326,7 +326,8 @@ class CircuitPolynomialBuilder {
   }
 
   void UpdateVanishingLookups(size_t circuit_idx) {
-    size_t num_lookups = committed_lookups_vec_->size();
+    CHECK_LT(circuit_idx, committed_lookups_vec_->size());
+    size_t num_lookups = (*committed_lookups_vec_)[circuit_idx].size();
```

Comment:
Can you make this change a separate commit or fix the commit message to include all these changes?

Reply:
@TomTaehoonKim `num_lookups` determines the number of iterations for the following for-loop.

Reply:
@chokobole I've checked my comment! You may resolve this after you check it as well.